src/Components/ServiceScene/Breakdowns/ClearFiltersLayoutScene.tsx

```diff
@@ -0,0 +1,24 @@
+import { SceneComponentProps, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { GrotError } from '../../GrotError';
+import { Alert, Button } from '@grafana/ui';
+import React from 'react';
+import { emptyStateStyles } from './FieldsBreakdownScene';
+
+export interface ClearFiltersLayoutSceneState extends SceneObjectState {
+  clearCallback: () => void;
+}
+export class ClearFiltersLayoutScene extends SceneObjectBase<ClearFiltersLayoutSceneState> {
```

Comment:
We could also make the message configurable and reuse here https://github.com/grafana/grafana/blob/main/public/app/features/trails/Breakdown/ByFrameRepeater.tsx#L163

Reply:
A bit confused, are you proposing we move this to core?